test/test_file.rb

```diff
@@ -47,7 +47,7 @@
     path = file.on_disk_path
 
     assert ::File.exist?(path)
-    assert_match(/^# Bilbo Baggins\n\nBilbo Baggins/, IO.read(path))
+    assert_match(/^# Bilbo Baggins[\n\r]+Bilbo Baggins/, IO.read(path))
```

Comment:
Will this test pass on unix-based systems?

Reply:
I think so, unless my regex is off this should allow either `\n` or `\r`.

Reply:
My bad! Looks fine.